### ISSUE TITLE
Is it possible to save the whole page ? 

### ISSUE THREAD

**Issue by leo3620** [wojtekmaj/react-pdf#829]
**Before you start - checklist**

 - [ x] I have read documentation in README
 - [x ] I have checked [sample](https://github.com/wojtekmaj/react-pdf/tree/master/sample) and [test](https://github.com/wojtekmaj/react-pdf/tree/master/test) suites to see real life basic implementation
 - [x ] I have checked if this question is not already asked

**What are you trying to achieve? Please describe.**
I try to save the full html page after it's rendered to send it to my c# backend to do some stuff. 

**Describe solutions you've tried**
Basically I tried to copy the html tag and save in a empty html file. When I open it I can see the transparent text but either the svg or canva are not rendered.

**Environment**

 - Browser (if applicable) [e.g. Chrome 57, Firefox 59]: Chrome
 - React-PDF version [e.g. 3.0.4]: v5.4.0-beta
 - React version [e.g. 16.3.0]: 17.0.2

Thank you !

**Comment by wojtekmaj** [comment]
Nope, it's not possible to just copy <canvas> as HTML. You could have more luck with SVG though.
